### ISSUE TITLE
Resolve Heroku deployment warning messages

### ISSUE THREAD

**Issue by ArthurZaharov** [fs/rails-base-graphql-api#36]
Warning messages occurs when we deploy the application to Heroku:
```
###### WARNING:

       You set your `config.active_storage.service` to :local in production.
       If you are uploading files to this app, they will not persist after the app
       is restarted, on one-off dynos, or if the app has multiple dynos.
       Heroku applications have an ephemeral file system. To
       persist uploaded files, please use a service such as S3 and update your Rails
       configuration.
       
       For more information can be found in this article:
         https://devcenter.heroku.com/articles/active-storage-on-heroku
       

###### WARNING:

       We detected that some binary dependencies required to
       use all the preview features of Active Storage are not
       present on this system.
       
       For more information please see:
         https://devcenter.heroku.com/articles/active-storage-on-heroku
       

###### WARNING:

       There is a more recent Ruby version available for you to use:
       
       2.6.6
       
       The latest version will include security and bug fixes, we always recommend
       running the latest version of your minor release.
       
       Please upgrade your Ruby version.
       
       For all available Ruby versions see:
         https://devcenter.heroku.com/articles/ruby-support#supported-runtimes

###### WARNING:

       No Procfile detected, using the default web server.
       We recommend explicitly declaring how to boot your server process via a Procfile.
       https://devcenter.heroku.com/articles/ruby-default-web-server
```

**Comment by EvgeniyEsaulkov** [comment]
I investigated these warnings. Below are my thoughts:

1. ```You set your `config.active_storage.service` to :local in production.``` 
As described in the related text uploaded files will eventually disappear (after app deploy or restart). As we don't use file uploading in this project it doesn't matter, I guess. But we can 'hack' this message by setting `config.active_storage.service = :test` for production environment.

2. ```We detected that some binary dependencies required to use all the preview features of Active Storage are not present on this system.```
By default Rails uses `poppler` for PDF previews, and `ffmpeg` for Video previews. At the same time these tools are not available by default on Heroku. To remove this warning we can add buildpack with these dependencies - https://github.com/heroku/heroku-buildpack-activestorage-preview. But I guess we don't need to install this just to avoid Heroku warning. If we're not going to use these preview features let's ignore it.

3. ```There is a more recent Ruby version available for you to use: 2.6.6```
This Ruby version includes two security fixes (https://www.ruby-lang.org/en/news/2020/03/31/ruby-2-6-6-released/). It's a single improvement by comparising with Ruby 2.6.5 that we use now.

4. ```No Procfile detected, using the default web server.```
It's a good practice to have Procfile for Heroku projects. Also we can specify release command in this file - it will allow us to not run migrations manually after deploy.

I suggest to implement points 3 and 4 (update Ruby and add Procfile). Warnings related to Active Storage could be ignored for now.

**Comment by ArthurZaharov** [comment]
@EvgeniyEsaulkov 
Let's fix the 3 and 4 warning messages within this issue then.
